src/components/articles/article-card.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @next/next/no-img-element */
+interface ArticleCardProps {
+  article: {
+    id: string;
+    createdAt: string;
+    updatedAt: string;
+    userId: string;
+    description: string | null;
+    title: string;
+    content: string;
+    cover: string | null;
+  };
+}
+
+export const ArticleCard = ({ article }: ArticleCardProps) => {
+  const formattedDate = article.createdAt
+    ? article.createdAt.slice(0, 10)
+    : "Unknown Date";
+  return (
+    <div className="rounded-xl shadow-md overflow-hidden my-4 w-full">
```

Comment:
Should not have margin. The margin should be controlled by the container using `gap-` as it is more flexible.